### ISSUE TITLE
How to update when changes are made

### ISSUE THREAD

**Issue by coderZalmy** [modded-ubuntu/modded-ubuntu#170]
How can I update the GUI etc. when changes have been made? Do I rerun one of the shell scripts? Do I need to redownload anything?

**Comment by BDhackers009** [comment]
We're using the Ubuntu stable versions. So just do ` sudo apt update && sudo apt upgrade -y`

**Comment by coderZalmy** [comment]
I mean when bug fixes are made. I. E. The one I posted about icon themes not working and panel not working with light themes.

**Comment by coderZalmy** [comment]
Here
https://github.com/modded-ubuntu/modded-ubuntu/issues/169#issue-2754927437

**Comment by BDhackers009** [comment]
Yeah then you should reinstall modded ubuntu to get the latest patches.

**Comment by coderZalmy** [comment]
So rerun a script, or do I have to remove the whole thing and start from the beginning?

**Comment by BDhackers009** [comment]
You can rerun the script but it could result as some new errors or duplicates. So the best way is to remove the modded ubuntu by running the remove.sh file then run the scripts again.

**Comment by coderZalmy** [comment]
Is there a way to backup certain data? I also removed firefox because I like to use Firefox Nightly instead. Maybe you can make an option to skip installing a browser (with a warning for non-advanced users)

**Comment by BDhackers009** [comment]
I think there is already an option to install browsers when running gui.sh

**Comment by coderZalmy** [comment]
I want to skip browser installation because it only shows Chrome Or Firefox stable builds

**Comment by BDhackers009** [comment]
I see. We will try to add this option in next update. Thanks for your suggestion.

**Comment by coderZalmy** [comment]
> Here [#169 (comment)](https://github.com/modded-ubuntu/modded-ubuntu/issues/169#issue-2754927437)

Also the themes issue

**Comment by BDhackers009** [comment]
Yeah we're trying to fix this issue. Please drop your telegram username. I'll text you when there is any new update so that you can test it and tell us if there is any new issue.

**Comment by coderZalmy** [comment]
https://t.me/coderZalmy

**Comment by BDhackers009** [comment]
> https://t.me/coderZalmy

t.me/bdhackers009

this is my only telegram account please don't reply anyone else who claims to come from my behalf.
